src/endpoints/authentication/impl.cpp

```diff
@@ -414,6 +407,9 @@ namespace irods::http::handler
 
 						int* correct{};
 
+                        // NOLINTNEXTLINE(cppcoreguidelines-owning-memory, cppcoreguidelines-no-malloc)
+                        irods::at_scope_exit free_memory{[&correct] { std::free(correct); }};
+
```

Comment:
indentation.  check your soft tabs or something... blaming the IDE...

Reply:
Yes, indentation is hideous. I'll need to either tweak some config or use clang-format(?).

@korydraughn not sure if clang-format is good to use now? Think we ignored it originally since we were playing with the config still?

Reply:
That's correct. We've been ignoring clang-format for now.

The plan was to run it one time across all source files so everything is updated in one commit. I am planning on going through the implementation and disabling clang-format for certain things since it can make code really ugly.

Tweak your editor's config to use whatever the project uses for indentation. You'll have to do that anyway if your editor is configured for spaces.

Reply:
After some discussion, we've decided that @MartinFlores751 should run clang-format on his PR.

We can adjust formatting later if needed.

Reply:
Formatted!